### PR TITLE
Communicate over MQTT.

### LINKED ISSUE #4: Communicate via MQTT

### PR DESCRIPTION
**Pull request by alexeisersun**
Closes #4

### CODE REVIEW

File: device/device.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    def run(self):
+        self.client.connect(self.broker_url)
+        print(f"Connect to {self.broker_url}")
+
+        self.client.loop_forever()
+        pass
+
+    def on_connect(self, client, userdata, flags, result_code):
+        print(f"Connected to {self.broker_url}")
+        payload = self.crypt.encrypt(b"HELLO")
```
Comment (strdr4605) [inline device/device.py]: Where is the payload used?

File: mobile_app.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "client_private": client_private_key,
+        "device_public": device_public_key,
+        **config,
+    }
+
+
+class MobileApp(object):
+    def __init__(self, configuration):
+        self.crypt = Crypt(
+            configuration["client_private"], configuration["device_public"]
```
Comment (strdr4605) [inline mobile_app.py]: Looks like this like in __init__ function is the only difference between MobileApp and Device. Is it possible to create a base class and pass the field device_public in __init__ for both classes?

File: device/device.py (hunk 1 above)
Comment (alexeisersun) [reply device/device.py]: Not used, thus removed.

File: mobile_app.py (hunk 2 above)
Comment (alexeisersun) [reply mobile_app.py]: I've rethought how this class is working. I'll inherit from `Device` and override `on_connect` for class' purposes.

**Pull request merged**